package.json

```diff
@@ -97,5 +97,9 @@
     },
     "jestSonar": {
         "reportPath": "reports/js"
+    },
+    "dependencies": {
+        "tiptap": "^1.32.1",
+        "tiptap-extensions": "^1.35.1"
```

Comment:
can we move this to dev dependencies?

Reply:
I agree we should move it! 

As for the field, I wasn't sure which one to do, but went directly inside the booking page so the user can see directly what is change will look like before sending it and navigating between pages anyway! 

If we want it inside the settings page, I don't mind I can make the switch this weekend